src/lib/fauna.mjs

```diff
@@ -0,0 +1,31 @@
+export const getV10Client = async ({ url, secret }) => {
+  const { Client } = (await import("fauna")).default;
+  return new Client({
+    secret,
+    endpoint: url,
+  });
+};
+
+export const runV10Query = async ({
+  query,
+  url = undefined,
+  secret = undefined,
+  client = undefined,
+  options = {
+    format: "simple",
+    typecheck: false,
+  },
+}) => {
+  let _client = client;
```

Comment:
is there any reason to do this indirection? we can just check if `client` is defined and use that name later.

Reply:
I did it this way so that I can easily close the client created internally if one is not provided.